src/fields/MultiSelect.tsx

```diff
@@ -188,6 +189,9 @@ const Box = styled.div<{
     }
 
     > .rs-picker-tag-wrapper {
+      .rs-tag {
```

Comment:
Haha I annoy you but I prefer that we use `>` because it makes selector targeting stronger (= optimize rendering + takes precedence over everything else). It also avoids side effects as well as the use of `!important` (which you didn't do here but that's just to explain "why").